Tests/FluentAssertions.Specs/Formatting/FormatterSpecs.cs

```diff
@@ -216,6 +216,119 @@ public void When_the_object_is_a_generic_type_without_custom_string_representati
             .WithMessage("*StuffId =*");
     }
 
+    [Fact]
+    public void When_the_object_is_a_user_defined_type_it_should_show_the_name_on_the_initial_line()
+    {
+        // Arrange
+        var stuff = new StuffRecord(42, "description", new(24), new List<int> { 10, 20, 30, 40 });
+
+        // Act
+        Action act = () => stuff.Should().BeNull();
+
+        // Assert
+        act.Should().Throw<XunitException>()
```

Comment:
🤔 Since these tests are all about formatting, shouldn't we use the literal output for the tests (as raw strings)? With C# 11, this is a bit easier as well. Also a question for @jnyrup

Reply:
I'm all in for upgrading to C# 11 and using raw string literals where applicable.

Reply:
Thanks, I've updated four of the tests to use raw string literals - it makes it so much clearer what the expected output is.

It has also highlighted another unintended change in behaviour - lists are not quite formatted the same as they used to be.  I ran the test `When_the_object_is_a_generic_type_without_custom_string_representation_it_should_show_the_properties` in `develop` to see the original expected output and it doesn't quite match the output produced by this branch.

I'm out of time this morning, I've pushed the branch even though that test doesn't pass, and I'll have a look at fixing it later tonight.

Enjoy the rest of your weekend!